GNNLux/src/layers/conv.jl

```diff
@@ -515,4 +515,60 @@ function Base.show(io::IO, l::GATv2Conv)
     l.σ == identity || print(io, ", ", l.σ)
     print(io, ", negative_slope=", l.negative_slope)
     print(io, ")")
+end
+
+@concrete struct SGConv <: GNNLayer
+    in_dims::Int
+    out_dims::Int
+    k::Int
+    use_bias::Bool
+    add_self_loops::Bool
+    use_edge_weight::Bool
+    init_weight
+    init_bias    
+end
+
+function SGConv(ch::Pair{Int, Int}, k = 1;
+                init_weight = glorot_uniform,
+                init_bias = zeros32,
+                use_bias::Bool = true,
+                add_self_loops::Bool = true,
+                use_edge_weight::Bool = false)
+    in_dims, out_dims = ch
+    return SGConv(in_dims, out_dims, use_bias, add_self_loops, use_edge_weight, init_weight, init_bias, k)
```

Comment:
the arguments passed to the constructor are not in the correct order. Please be more careful before committing and run tests locally. I cannot go over every tiny detail.

Reply:
will check